catalog/ui/search-ui/search-endpoint/src/test/java/org/codice/ddf/ui/searchui/query/controller/SearchControllerTest.java

```diff
@@ -396,10 +460,71 @@ private CatalogFramework createFramework() {
         return framework;
     }
 
+    private void runTestForReasonMessages() throws Exception {
```

Comment:
I'll probably remove these overloaded methods. Originally I wanted to use them as neat defaults for testing assertions that should result in an empty list, which means no exception was thrown and the query was successful on all levels of service. However, there is coupling between `SearchController` and two runnable classes that make this not feasible. 

In a typical design, we should let **either** the controller **or** the runnable classes make calls to `search.publishResults` regardless of success or failure. This is not the case, so I am stuck only able to test the failure scenarios.